common/lib/capa/capa/templates/annotationinput.html

```diff
@@ -59,6 +60,5 @@
 </form>
 
 % if msg:
-<span class="message">${msg|n}</span>
+<span class="message">${HTML(msg)}</span>
```

Comment:
I made a mistake. Using HTML() here is fine if msh is always HTML.

If the message could be HTML or plain text, you have two options:
1. HTML-escape the plain text before it gets here so it too is HTML by the time you get here, or
2. Wrap the HTML with HTML() before you get here, if it wasn't already created in such a way that it was marked as markup. Then, let the default escaping apply in the mako template which will only apply to the plain text.